tests/integration_test/__tests__/mass-upload.test.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+/* eslint-disable no-sync */
+
+const fs = require('fs');
+const crypto = require('crypto');
+const utils = require('../lib/utils');
+const systemPath = require('path');
+const { getAppClient, getUserClient } = require('../context');
+const { createBoxTestFolder } = require('../objects/box-test-folder');
+const { createBoxTestUser, clearUserContent } = require('../objects/box-test-user');
+const context = {};
+
+const CHUNKED_UPLOAD_MINIMUM = 20000000;
+
+beforeAll(async() => {
+	let appClient = getAppClient();
+	let user = await createBoxTestUser(appClient);
+	let userClient = getUserClient(user.id);
+	let folder = await createBoxTestFolder(userClient);
```

Comment:
if you define these outside `beforeAll` you'll be able to access them in tests and won't need "context"

Reply:
what about this?